web/ai_service.py

```
@@ -0,0 +1,141 @@
+            'markdown.extensions.attr_list',
+            'markdown.extensions.def_list',
+            'markdown.extensions.footnotes'
+        ])
+        
+        # Add custom CSS classes for better styling
+        html = html.replace('<code>', '<code class="inline-code">')
+        html = html.replace('<pre>', '<pre class="code-block">')
+        
+        return html
+
+    def get_response(self, prompt: str, context: Optional[Dict[str, Any]] = None) -> str:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)